dev/packages/beats/cisco-0.0.1/dataset/ios/agent/stream/pipeline.js

```diff
@@ -0,0 +1,222 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+var ciscoIOS = (function() {
```

Comment:
No need to answer this in this PR but I wonder how this will work. I assume at the moment Filebeat will still ship with this. I assume at the moment this should not be in the package.